src/components/CalloutCard/CalloutCard.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './CalloutCard.scss';
+
+const CalloutCard = ( { children } ) => (
+	<div className={styles.container}>
+		{children}
+	</div>
+);
+
+CalloutCard.propTypes = {
+	/**
+	 * Child nodes to be rendered as the inner content of the callout card.
+	 */
+	children: PropTypes.oneOfType( [
+		PropTypes.arrayOf( PropTypes.node ),
```

Comment:
Interesting, do you get a proptype error without this?

Reply:
Oh I guess not - I'll simplify to node.